schema

```diff
@@ -1 +1 @@
-Subproject commit c4b69c3a5076d31e2d40d5ab7f053aab6bdbe938
```

Comment:
You should be able to get rid of this change by doing:

```
rm -rf schema/
git submodule init
git submodule update
```

And then committing the result.